bin/index_db.sh

```diff
@@ -287,6 +291,21 @@ if [ "$data_only" == "true" ]; then
     echo "$CREATE_MODEL" >> create_models.job.sh
 else
   eval "$CREATE_MODEL"
+
+  if ["$train_llm" == "true"]; then
+    echo "Training LLM Weights"
+    echo "Downloading ranklib..."
+    mkdir -p $BASE_WDIR/ranklib/
+    cd $BASE_WDIR/ranklib/
+    curl -L -o RankLib-2.1-patched.jar http://downloads.sourceforge.net/project/lemur/lemur/RankLib-2.1/RankLib-2.1-patched.jar?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Flemur%2Ffiles%2Flemur%2FRankLib-2.1%2F&ts=1439317425&use_mirror=skylink
```

Comment:
Yeah, might be nice to do that! On the other hand it's only a training-time dependency, right?